packages/design-system/src/components/InlineEditing/Primitives/InlineEditingPrimitive.tsx

```diff
@@ -142,8 +141,17 @@ const InlineEditingPrimitive = forwardRef(
 			onChange: (
 				event: React.ChangeEvent<HTMLInputElement> | React.ChangeEvent<HTMLTextAreaElement>,
 			): void => setValue(event.target.value),
+			// Keyboard shortcuts
+			onKeyDown: (event: React.KeyboardEvent) => {
+				if (event.keyCode === 13 && mode !== 'multi') {
```

Comment:
I'm surprised to not find any enum of keyCode anywhere easily on a npm package, so let's go for the `13` stuff.
Could we update it using a const somewhere ?

Reply:
We use [keycode](https://www.npmjs.com/package/keycode) in other packages, maybe it can be used here too :)